rustls/src/client/client_conn.rs

```diff
@@ -246,6 +246,15 @@ impl ClientConfig {
         Self::builder_with_provider(crate::crypto::ring::RING)
     }
 
+    #[cfg(all(not(feature = "ring"), feature = "aws_lc_rs"))]
+    /// Create a builder to build up the client configuration with the default
+    /// [`CryptoProvider`].
+    ///
+    /// For more information, see the [`ConfigBuilder`] documentation.
+    pub fn builder() -> ConfigBuilder<Self, WantsCipherSuites> {
```

Comment:
It's hard to articulate, but describing this as the "default CryptoProvider" feels a little bit confusing. I guess I'm assuming we'd only have one of the `ring` or `aws_lc_rs` features enabled by default right?

Since this is conditional on the *ring* feature being absent and the aws_lc_rs feature being present maybe it would be less confusing to explicitly say it will use aws-lc-rs.

Reply:
> I'm assuming we'd only have one of the ring or aws_lc_rs features enabled by default right?

Maybe a table would make the intention clearer:

| Features | What provider backs `ClientConfig::builder()` ||
|-|-|-|
| _none_ | Absent ||
| `ring` | `ring::RING` | <- crate default |
| `aws_lc_rs` | `aws_lc_rs::AWS_LC_RS` ||
| `aws_lc_rs`, `ring` | `ring::RING` |

Reply:
> Since this is conditional on the ring feature being absent and the aws_lc_rs feature being present maybe it would be less confusing to explicitly say it will use aws-lc-rs.

Done that (for all four `builder()` functions)

Reply:
Thanks, I think this is clearer now.